lib/webpack.js

```diff
@@ -15,7 +16,9 @@ function webpack(options, callback) {
 			return webpack(options);
 		}));
 	} else if(typeof options === "object") {
-		if(!options.entry && !options.plugins) {
+		var webpackOptionsValidationErrors = validateWebpackOptions(options);
+		if(webpackOptionsValidationErrors.length) {
+			console.log("'options' validation errors:", webpackOptionsValidationErrors);
```

Comment:
You are only allowed use console.log in the CLI not in the API.

Instead rewrite the code to throw a CustomError with a `validationErrors` property. Add code to the CLI to check for this property and format the errors.